build/ProjectReunion-release.yml

```diff
@@ -0,0 +1,206 @@
+#n.b. this is a parameter in WinUI-BuildMUXProject-Steps.yml  
+  buildArtifactsDirectory: $(Build.SourcesDirectory)\BuildOutput
+  
+jobs:
+
+- job: Build
+  # Skip the build job if we are reusing the output of a previous build.
+  # useBuildOutputFromBuildId variable is set on the Pipeline at Queue time.
+  condition:
+    eq(variables['useBuildOutputFromBuildId'],'')
+  pool:
+    vmImage: 'windows-2019'
+  timeoutInMinutes: 120
+  strategy:
+    maxParallel: 10
+    matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+        normalizedConfiguration: 'fre'
+        PGOBuildMode: 'Optimize'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+        normalizedConfiguration: 'fre'
+        PGOBuildMode: 'Optimize'
+      Release_Arm:
+        buildPlatform: 'arm'
+        buildConfiguration: 'Release'
+        normalizedConfiguration: 'fre'
+      Release_Arm64:
+        buildPlatform: 'arm64'
+        buildConfiguration: 'Release'
+        normalizedConfiguration: 'fre'
+
+  variables:
+    appxPackageDir : $(build.artifactStagingDirectory)\$(buildConfiguration)\$(buildPlatform)\AppxPackages
+    buildOutputDir : $(Build.SourcesDirectory)\BuildOutput
+    publishDir : $(Build.ArtifactStagingDirectory)
+  steps:
+
+  # Download and extract nuget package with non-stubbed MicrosoftTelemetry.h header
+#  - task: DownloadPackage@1
+#    displayName: 'Download Microsoft.Telemetry.Inbox.Native'
+#    inputs:
+#      feed: '/3415933f-ac0d-4766-8c0a-3f4c247c25f5'                         # 0
+#      view: 'ef61a1c1-003b-4a27-bde5-beec8301021b'                          # Release
+#      definition: '2fe60c09-c66f-4275-ae2d-f015c7170c72'                    # Microsoft.Telemetry.Inbox.Native
+#      version: '10.0.18362.1-190318-1202.19h1-release.amd64fre'             # latest version
+#      downloadPath: '$(System.DefaultWorkingDirectory)'                     # download and extract to repo root
+
+  # Replace the stubbed MicrosoftTelemetry.h with the real one
+  # Delete the existing stubbed MicrosoftTelemetry.h first, to ensure that if it is no longer at the expected path that the task, and build, fails
+#  - script: |
+#     del $(System.DefaultWorkingDirectory)\dev\telemetry\MicrosoftTelemetry.h
+#     move /Y $(System.DefaultWorkingDirectory)\build\native\inc\MicrosoftTelemetry.h $(System.DefaultWorkingDirectory)\dev\telemetry\
+#    failOnStderr: true
+#    displayName: 'Replace existing stubbed MicrosoftTelemetry.h header with the real version from the nuget package'
+
+  - template: AzurePipelinesTemplates\ProjectReunion-BuildDevProject-Steps.yml
+
+  - template: AzurePipelinesTemplates\ProjectReunion-PublishProjectOutput-Steps.yml
+
+# component detection must happen *within* the build task
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+
```

Comment:
This needs to be part of build steps. We don't want someone brings in something breaking this, and don't know that until we need to release.

Reply:
I thought we covered this in email.  I read the collateral carefully, this is the last step in the build job, which is later than requested by the signing team, but it is the place that covers the most things that could be introduced prior to governance detection.  This was verified as correct by Daniel Choi before approving signing.

Reply:
My comment is not about order. We need that task to be part of CI/PR. Currently it's only run in the release pipeline, but the release pipeline is not gated for CI/PR.

Reply:
Ah, thank you for clarifying.  No one else I looked at runs a governance task in their CI pipeline.  They detect this when they try to release.  I suppose that is pretty late.  Let me log an issue about this and we can discuss.

Reply:
MRTCore runs it. :) We don't want to be blocked at release and find it may not be easy to fix.

Reply:
https://github.com/microsoft/ProjectReunion/issues/225

Reply:
:-) We can look into doing this when the dust settles (esp. with MRTCore integration into mainline build/release).  You can feel free to pick this up if you have spare cycles.  Otherwise we will get this scheduled.

Reply:
according to https://osgwiki.com/wiki/Certificates_available_for_Package_ES_CodeSign. Most signing uses cert 400. It's not clear to me when WindowsRT needs to be used.

Reply:
I took this to email, I'll report back here with what we find out.

Reply:
The 444 'WindowsRT' cert covers a broader range of use cases so it is recommended to use it over the 400 cert.